gamification/src/Components/GradeStudent/GradeStudent.js

```diff
@@ -12,24 +12,71 @@ function GradeStudent() {
         </Row>
         <Row className="mt-2">
           <Col lg={12}>
-            <h3>Calificando a: {`nombre`}</h3>
+            <h3>Calificando a: {`nombre y matrícula del alumno`}</h3>
           </Col>
         </Row>
 
-        <Row className="mt-3">
-          <Col>
-            <p>Form aqui</p>
-          </Col>
-        </Row>
-        <Row className="mt-3">
-          <Col>
-            <Button variant="danger">Cancelar</Button>
-          </Col>
-          <Col>
-            {' '}
-            <Button variant="success">Registrar</Button>
-          </Col>
-        </Row>
+        <Form>
+          <Row className="mt-3">
+            <Col lg={6}>
+              <h3>Misión</h3>
+              <Form.Select aria-label="Default select example">
+                <option>Misión a calificar</option>
+                <option value="1">One</option>
+                <option value="2">Two</option>
+                <option value="3">Three</option>
+              </Form.Select>
```

Comment:
Add todo to use react hooks that change the original value of the mission given so that it can be changed based on merit without professor needing to lookup value